templates/Gemfile.erb

```diff
@@ -25,9 +25,8 @@ gem "tzinfo-data", platforms: [:mingw, :x64_mingw, :mswin, :jruby]
 gem "webpacker"
 
 group :development do
-  gem "listen"
   gem "rack-mini-profiler", require: false
-  gem "spring"
```

Comment:
I think the `spring` entry shouldn't have been removed here.

If I created a project using Suspenders, and then chose to remove `spring-watcher-listen` from my Gemfile, it would would cause Spring to removed, even thought that wasn't the intention.

Reply:
That makes sense to me. We can add it back.